src/test/java/parser/ParserTest.java

```diff
@@ -0,0 +1,43 @@
+package parser;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class ParserTest {
+
+    @Test
+    public void isValidDateTest1() {
```

Comment:
Minor nitpick - maybe some comments to describe what the test is could be useful? Alternatively, consider the naming convention `unitBeingTested_descriptionOfTestInputs_expectedOutcome` for method names to make them self-explanatory (from week 3 topics).